mitosheet/src/components/footer/Footer.tsx

```diff
@@ -46,13 +49,28 @@ function Footer(props: FooterProps): JSX.Element {
                 <PlusIcon/>
             </div>
             <div className="footer-tab-bar">
+                {/* First add the data tabs, and then add the graph tabs */}
                 {props.sheetDataArray.map(df => df.dfName).map((dfName, idx) => {
                     return (
                         <SheetTab
```

Comment:
Sheet Tab feels like the wrong name for this thing now

Reply:
To me it reflects the model that makes more sense. We have two different types of sheets -- one for data and one for graphs. But currently, its weird because the graph sheet is hacked to actually just be a large taskpane.